autogen/AudioSource.m

```diff
@@ -28,13 +29,18 @@
     end
     
     methods
-        function obj = AudioSource(audio, sampleRateHz)
+        function obj = AudioSource(audio, sampleRateHz, dataType)
         %AudioSource Initializes AudioSource with m x n audio input and
         % sample rate in hz, where m is the the number of samples and n is
         % the number of channels.
         
             obj.sampleRateHz = sampleRateHz;
             obj.audio = audio;
+            if exist('dataType','var')
+                obj.dataType = dataType;
+            else
+                obj.dataType = fixdt(1,24,23);
+            end
```

Comment:
If we wanted, this could be handled by setting a default value in an `arguments` block. I've grown quite fond of the `arguments` block. However, doing that is not really any less boilerplate than the current implementation.